smoke-tests/apps/JavaProfiler/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/JavaProfilerEnabledTest.java

```diff
@@ -59,9 +59,9 @@ static class Java11Test extends JavaProfilerEnabledTest {}
   @Environment(JAVA_17)
   static class Java17Test extends JavaProfilerEnabledTest {}
 
-  @Environment(JAVA_20)
-  static class JavaLatestTest extends JavaProfilerEnabledTest {}
-
   @Environment(JAVA_21)
-  static class JavaPrereleaseTest extends JavaProfilerEnabledTest {}
+  static class Java21Test extends JavaProfilerEnabledTest {}
+
+  @Environment(JAVA_21_OPENJ9)
+  static class Java21OpenJ9Test extends JavaProfilerDisabledTest {}
```

Comment:
the profiler doesn't work on OpenJ9 which is why OpenJ9 wasn't tested here previously